tests/e2escenarios/e2e_devfile_test.go

```diff
@@ -0,0 +1,174 @@
+package e2escenarios
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift/odo/tests/helper"
+
+	. "github.com/onsi/ginkgo"
+)
+
+// Test Objective:
+//    Test ODO devfile support features
```

Comment:
Do we really need this. AFAIU Ginkgo already provides such features of describing the test objective like https://github.com/openshift/odo/pull/4179/files#diff-6006c13a40a158e7211b40a7d4fa9ed0d34339c33f32459b97f9c8a22ead95adR24

Reply:
By adding this at the top of the script it provides a quick way, for anyone making changes or additions, to get a high level picture of the scope of the script. Otherwise, whoever is making changes will have to scan through the code. Basically this reduces the time to understand what the code does from minutes to seconds. I understand that that the describe, context and it provides that to some degree, however the text in there tends to be limited. saving time is always welcome unless you disagree.